README.md

```diff
@@ -1,48 +1,105 @@
-<!--
+# Penpot Charm
+
 [![CharmHub Badge](https://charmhub.io/penpot/badge.svg)](https://charmhub.io/penpot)
 [![Publish to edge](https://github.com/canonical/penpot-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/penpot-operator/actions/workflows/publish_charm.yaml)
 [![Promote charm](https://github.com/canonical/penpot-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/penpot-operator/actions/workflows/promote_charm.yaml)
 [![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
--->
-
-A Juju charm that deploys and manages [Penpot](https://penpot.app) on Kubernetes. Penpot is the
-web-based open-source design tool that bridges the gap between designers and
-developers.
-
-<!--
-This charm simplifies initial deployment and "day N" operations of Penpot,
-such as scaling the number of instances, integration with external
-authentication providers, access to S3 for redundant file storage and more. It
-allows for deployment on many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
-[Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
-offerings.
--->
-
-As such, the charm makes it easy for those looking to take control of their
-own Penpot deployment while keeping operations simple, and gives them the
-freedom to deploy on the Kubernetes platform of their choice.
-
-<!--
-For DevOps or SRE teams this charm will make operating Penpot simple and
-straightforward through Juju's clean interface. It will allow easy deployment
-into multiple environments for testing of changes, and supports scaling out for
-enterprise deployments.
--->
 
-## Project and community
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+for deploying and managing the [penpot](https://penpot.app) open-source
+design tool for design and code collaboration in your systems.
+
+This charm simplifies the configuration and maintenance of penpot across a 
```

Comment:
Same here, wherever the application name is used. For charm- you can keep it in lowercase